js/templates/ratingStars.html

```diff
@@ -1,15 +1,3 @@
-<% for (var i=0; i < ob.rating; i++) { %>
```

Comment:
The reason I had the template the previous way was to support partial stars. For example, when we calculate the summary review at the top of an item as an average of all the individual reviews, the figure is often times a decimal (e.g. 3.36). In that case, I believe we were rounding to the nearest quarter percent (e.g. 3.25) and then showing 3 full stars and 1 quarter star.

I believe this adjusted code, while being much simpler, will not support that. Have we decided that we don't need to support partials stars?